caravel/assets/spec/javascripts/sqllab/QuerySearch_spec.jsx

```diff
@@ -15,7 +15,7 @@ describe('QuerySearch', () => {
 
   it('should have two Select', () => {
     const wrapper = shallow(<QuerySearch />);
-    expect(wrapper.find(Select)).to.have.length(2);
+    expect(wrapper.find(Select)).to.have.length(4);
```

Comment:
hmm by state mutations do you mean tests for reducers? for now we haven't had much tests for functionalities, most are for rendering components in javascript